run-tests.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# -*- coding: utf-8 -*-
+#
+# This file is part of Cookiecutter - Invenio Module Template
+# Copyright (C) 2017 ETH Zurich, Swiss Data Science Center.
+#
+# Cookiecutter - Invenio Module Template is free software; you can
+# redistribute it and/or modify it under the terms of the GNU General Public
+# License as published by the Free Software Foundation; either version 2 of
+# the License, or (at your option) any later version.
+#
+# Cookiecutter - Invenio Module Template is distributed in the hope that it
+# will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Cookiecutter - Invenio Module Template; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# 02111-1307, USA.
+
+# quit on errors:
+set -o errexit
+
+# quit on unbound symbols:
+set -o nounset
+
+WORKDIR="`mktemp -d`"
```

Comment:
Thanks, there are two more changes:

- `$(mktemp -d)`
- ` rm -rf "${WORKDIR}"`

see:

```console
$ shellcheck ./run-tests.sh 

In ./run-tests.sh line 28:
WORKDIR="`mktemp -d`"
         ^-- SC2006: Use $(..) instead of legacy `..`.


In ./run-tests.sh line 33:
    rm -rf ${WORKDIR}
           ^-- SC2086: Double quote to prevent globbing and word splitting.


In ./run-tests.sh line 46:
pip install -e .[all] --quiet
                ^-- SC2102: Ranges can only match single chars (mentioned due to duplicates).

```
```